pkg/kubelet/token/token_manager.go

```diff
@@ -147,5 +148,16 @@ func (m *Manager) requiresRefresh(tr *authenticationv1.TokenRequest) bool {
 
 // keys should be nonconfidential and safe to log
 func keyFunc(name, namespace string, tr *authenticationv1.TokenRequest) string {
-	return fmt.Sprintf("%q/%q/%#v", name, namespace, tr.Spec)
+	if tr.Spec.ExpirationSeconds == nil {
```

Comment:
These two if statements are unneccesary and they modify the input which is strange for a keyfunc. Can you just do the single fmt.Sprintf in here?